doc/contributing.rst

```diff
@@ -721,7 +721,14 @@ Perform the steps listed below with two persons, to reduce the risk of error.
    `PyPI <https://pypi.org/project/ESMValCore/>`__, and
    `readthedocs <https://readthedocs.org/dashboard/esmvalcore/users/>`__.
 
-To make a new release of the package, follow these steps:
+The release of ESMValCore is tied to the release of ESMValTool. 
+To start the procedure, ESMValCore gets released as a 
+release candidate to test the recipes in ESMValTool. If bugs are found
+during the testing phase of the release candidate, make as many release 
```

Comment:
```suggestion
during the testing phase of the release candidate, and/or a number of release-worthy developments are included in the main branch during the release process, make as many release 
```

Reply:
I am not sure about this because then it would not be worth it to have a code freeze. I think there should be one, and what should be avoided is what happened in the last release not the other way around.